tasks/create.json

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
Let's change the `data` param to `fields`. Latter matches ServiceNow's docs. See "Request body"'s docs for `POST /now/table/{tableName}` in https://docs.servicenow.com/bundle/orlando-application-development/page/integrate/inbound-rest/concept/c_TableAPI.html#c_TableAPI

Reply:
Ditto for the `update` task.